src/main/java/uk/gov/hmcts/reform/hmc/model/linkedhearinggroup/LinkedHearingGroupResponses.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.hmc.model.linkedhearinggroup;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+public class LinkedHearingGroupResponses {
+
+    private GroupDetails groupDetails;
```

Comment:
groupname and linkedcomments appear as null in the response. Happens with a null check as well. Add @jsoninclude...